index.js

```diff
@@ -51,33 +53,39 @@ const styles = StyleSheet.create({
   bottomPicker: {
     width: SCREEN_WIDTH,
   },
-});
+};
 
 const propTypes = {
   buttonColor: PropTypes.string,
   buttonStyle: PropTypes.object,
-  options: PropTypes.array.isRequired,
-  initialOptionIndex: PropTypes.number,
-  labels: PropTypes.array,
-  confirmText: PropTypes.string,
   cancelText: PropTypes.string,
+  confirmText: PropTypes.string,
+  disableOverlay: PropTypes.bool,
+  initialOptionIndex: PropTypes.number,
   itemStyle: PropTypes.object,
+  labels: PropTypes.array,
+  modalVisible: PropTypes.bool,
   onSubmit: PropTypes.func,
-  disableOverlay: PropTypes.bool,
+	options: PropTypes.array.isRequired,
+	styles: PropTypes.object,
```

Comment:
Can you please format these lines?

Reply:
@syzer we have a [editorconfig](https://github.com/puredazzle/react-native-simple-picker/blob/master/.editorconfig) in the project, please try to keep the styling we have. thanks in advance.